src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -230,10 +232,19 @@ public bool TryGet<T>(StringSegment name, out T obj)
                 obj = default(T);
                 return false;
             }
-            ConvertType(result, out obj);
+            try
+            {
+                ConvertType(result, out obj);
+            }
+            catch
```

Comment:
I think, that if we failed to get a member with the specific type we should return false and not throw an error.